Source/Core/DolphinQt/Achievements/AchievementsWindow.cpp

```diff
@@ -28,6 +30,14 @@ void AchievementsWindow::CreateMainLayout()
 {
   auto* layout = new QVBoxLayout();
 
+  m_tab_widget = new QTabWidget();
+  m_tab_widget->addTab(
+      GetWrappedWidget(new AchievementSettingsWidget(m_tab_widget, this), this, 125, 100),
+      tr("Settings"));
+
+  m_button_box = new QDialogButtonBox(QDialogButtonBox::Close);
```

Comment:
This line needs to be moved to the first commit, otherwise `m_button_box` is uninitialized there.

(Or just squash the two commits, I think there's not much point separating these two here.)